hu/admin/user.php

```diff
@@ -9,10 +9,10 @@
 
 return [
     'exceptions' => [
-        'user_has_servers' => 'Cannot delete a user with active servers attached to their account. Please delete their servers before continuing.',
+        'user_has_servers' => 'Nem törölhető olyan felhasználó akinek karakteréhez működő szerver van csatolva. Kérlek töröld előtte az adott szervert mielőtt folytatnád.',
```

Comment:
Talán jobban érthető?
```suggestion
        'user_has_servers' => 'Nem törölhető olyan felhasználó, akinhez működő szerver van csatolva. Kérlek előbb töröld a felhasználóhoz csatolt szervereket.',
```